packages/vector_graphics/lib/src/_debug_web.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Skip rasterization on web platform debug mode.
+bool get debugSkipRaster {
+  return true;
```

Comment:
This seems likea bad default until we have web test users. 

Maybe use the zone thing for now?

Reply:
I'd rather leave it on then use Zones for anything